ShapeCrawler/AutoShapes/IParagraph.cs

```diff
@@ -9,6 +9,11 @@ namespace ShapeCrawler
     /// </summary>
     public interface IParagraph
     {
+        /// <summary>
+        /// Add text to paragraph.
+        /// </summary>
+        void AddText(string text);
```

Comment:
@sergsuhanov, thank you for the PR.

What about moving this to `IPortionCollection` level? I think something like `newPr.Portions.Add("Text2")` would be more consistent.

Reply:
hi, maybe
IPortion maps to A.Run and when we add a text separator we use A.Break.

`var lastElement = this.AParagraph.Where(p => p is A.Run || p is A.Break).Last(); // for inset after this`
I think that at the paragraph level, working with text is more logical.

Reply:
Got it. Then what about rename `AddText()` -> `AddPortion()`

Reply:
Ok, sure